src/Sylius/Bundle/ResourceBundle/Form/EventSubscriber/AddCodeFormSubscriber.php

```diff
@@ -51,15 +51,17 @@ public static function getSubscribedEvents()
     public function preSetData(FormEvent $event)
     {
         $resource = $event->getData();
+        $disabled = false;
 
-        if (!$resource instanceof CodeAwareInterface) {
+        if (!$resource instanceof CodeAwareInterface && isset($resource)) {
```

Comment:
Yes, because when OptionValue is creating (and it implements CodeAwareInterface) and we try add to this code field we get in ``$resource`` null, but we should get resource which implements CodeAwareInterface. For this reason we check if ``$resource`` is not null.